core/src/main/scala/com/salesforce/op/OpWorkflowModelWriter.scala

```diff
@@ -59,11 +57,36 @@ class OpWorkflowModelWriter(val model: OpWorkflowModel) extends MLWriter {
 
   implicit val jsonFormats: Formats = DefaultFormats
 
+  protected var modelStagingDir: String = WorkflowFileReader.modelStagingDir
+
+  /**
+   * Set the local folder to copy and unpack stored model to for loading
+   */
+  def setModelStagingDir(localDir: String): this.type = {
+    modelStagingDir = localDir
+    this
+  }
+
   override protected def saveImpl(path: String): Unit = {
-    JobGroupUtil.withJobGroup(OpStep.ModelIO) {
-      sc.parallelize(Seq(toJsonString(path)), 1)
-        .saveAsTextFile(OpWorkflowModelReadWriteShared.jsonPath(path), classOf[GzipCodec])
-    }(this.sparkSession)
+    val localPath = new Path(modelStagingDir)
+    val conf = new Configuration()
+    val localFileSystem = FileSystem.getLocal(conf)
+    if (shouldOverwrite) localFileSystem.delete(localPath, true)
```

Comment:
I think this line is no longer necessary now that we have `val out = localFileSystem.create(new Path(jsonPath), shouldOverwrite)` a bit below.

Reply:
i think we should always do `localFileSystem.delete(localPath, true)` irrespective of `shouldOverwrite`. shouldOverwrite should be applicable to the remote path i.e the passed to `saveImpl` fn.

Reply:
👍 Right, they're independent